buildSrc/src/main/java/saros/gradle/intellij/SarosIntellijPlugin.java

```diff
@@ -0,0 +1,46 @@
+package saros.gradle.intellij;
+
+import org.gradle.api.Plugin;
+import org.gradle.api.Project;
+import org.gradle.api.Task;
+import org.gradle.api.tasks.TaskContainer;
+import org.jetbrains.intellij.IntelliJPlugin;
+
+/**
+ * Plugin that configures the gradle intellij plugin. The used plugin version is specified by the
+ * build.gradle file in the buildSrc dir.
+ *
+ * <p>The main purpose of this plugin is to set default configs and to add additional tasks to the
+ * runIde task (provided by the intellij plugin) in order to allow us to run multiple intellij
+ * instances simultaneously with runIde. Without our custom tasks the second runIde call would
+ * fails, because the config directory (within the build dir of the corresponding project) is
+ * already in use. Therefore we enhance the config directory selection. See {@code
+ * ConfigureIntelliJTask} for more information.
+ */
+public class SarosIntellijPlugin implements Plugin<Project> {
+  @Override
+  public void apply(Project project) {
```

Comment:
Maybe apply also has a valid javadoc provided by the gradle library, but my IntelliJ instance can't resolve it. So it would be nice to have any indication what the `apply(Project)` method is used for.

Another solution would be to link the sources so that we can access the library javadoc.